dataactbroker/check_status.yml

```diff
@@ -0,0 +1,127 @@
+swagger: '2.0'
```

Comment:
Was the intent to check this into the repo? At some point, it would be great to use Swagger to replace our non-machine-readable API docs, but this looks like the file used to verify the change with the front-end team?